src/pages/[platform]/start/getting-started/setup/index.mdx

```diff
@@ -665,7 +665,17 @@ Next, configure the Amplify libraries client-side so it can interact with backen
         Open **src/main.js** and add the following code below the last import:
     </InlineFilter>
 
-    <InlineFilter filters={["angular", "react", "vue"]}>
+    <InlineFilter filters={["angular"]}>
+
+    ```javascript title="src/main.ts"
+    import { Amplify } from 'aws-amplify';
+    import amplifyconfig from './amplifyconfiguration.json';
+    Amplify.configure(amplifyconfig);
```

Comment:
```suggestion
    import outputs from './amplify_outputs.json';
    Amplify.configure(outputs);
```

the PR with this change in other examples hasn't been merged just yet https://github.com/aws-amplify/docs/pull/7346